templates/index.html

```diff
@@ -34,6 +35,7 @@
             <td>{{ resource.allocated }}</td>
             <td>{{ resource.allocated_to_id }}</td>
             <td>{{ resource.allocated_to_address }}</td>
+            <td>{{ resource.allocated_at }}</td>
```

Comment:
Not now, that would mean integrating a javascript library in the app. 

Can't be asked